zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/batchoperations/BatchOperationUpdateTask.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.tasks.batchoperations;
+
+import io.camunda.exporter.tasks.BackgroundTask;
+import io.camunda.exporter.tasks.batchoperations.BatchOperationUpdateRepository.DocumentUpdate;
+import io.camunda.webapps.schema.descriptors.operate.template.BatchOperationTemplate;
+import io.camunda.webapps.schema.entities.operation.BatchOperationEntity;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+
+public class BatchOperationUpdateTask implements BackgroundTask {
+
+  private final BatchOperationUpdateRepository batchOperationUpdateRepository;
+
+  private final Logger logger;
+
+  public BatchOperationUpdateTask(
+      final BatchOperationUpdateRepository batchOperationUpdateRepository, final Logger logger) {
+    this.batchOperationUpdateRepository = batchOperationUpdateRepository;
+    this.logger = logger;
+  }
+
+  @Override
+  public CompletionStage<Integer> execute() {
+
+    final var batchOperations = batchOperationUpdateRepository.getNotFinishedBatchOperations();
+
+    if (batchOperations.size() > 0) {
```

Comment:
use !isEmpty instead. I would prefer a short circuit where if the `batchOperations` is empty we return `CompletedFuture.completeFuture` as this makes things easier to read with less indentation

Reply:
Did that in the next PR: https://github.com/camunda/camunda/pull/25743/files#diff-51b7f2ad9c4e91779d547c372e43518ec31630e171bdff563bf3932dab78fcd3R37